### PR TITLE
Feature/lxl 4149 title filters

### PR DESCRIPTION
**Pull request by kwahlin**
See LXL-4149 for more info.

### CODE REVIEW

File: librisworks/src/main/groovy/se/kb/libris/mergeworks/Doc.groovy [hunk 1]
```diff
@@ -209,7 +209,10 @@ class Doc {
 
     boolean hasPart() {
         workData['hasPart'] || instanceData['hasTitle'].findAll { it['@type'] == 'Title' }.any {
-            it.hasPart?.size() > 1 || it.hasPart?.any { p -> asList(p.partName).size() > 1 || asList(p.partNumber).size() > 1 }
+            it.hasPart?.size() > 1
+                    || it.hasPart?.any { p -> asList(p.partName).size() > 1
+                    || asList(p.partNumber).size() > 1 }
+                    || [it.mainTitle, it.titleRemainder, it.subtitle].findAll().toString() =~ /(?<!\/.+)(\s+;)/
```
Comment (olovy) [inline librisworks/src/main/groovy/se/kb/libris/mergeworks/Doc.groovy:215]: Why `\/` in the regexp?

Comment (kwahlin) [reply librisworks/src/main/groovy/se/kb/libris/mergeworks/Doc.groovy:215]: We allow semicolon when it's preceded by a slash (not aggregate).

Comment (olovy) [inline librisworks/src/main/groovy/se/kb/libris/mergeworks/Doc.groovy:215]:
```suggestion
                    // space+semicolon indicates an aggregate if it is no preceded by a slash
                    // aggregate: Måsen ; Onkel Vanja ; Körsbärsträdgården
                    // not aggregate: En visa för de döda / Patrick Dunne ; översättning: Hans Lindeberg
                    || [it.mainTitle, it.titleRemainder, it.subtitle].findAll().toString() =~ /(?<!\/.+)(\s+;)/
```

**Review by olovy — APPROVED** [review]
LGTM!

See suggestion

**Review by niklasl — APPROVED** [review]
LGTM (and :+1:  on the suggested clarification)!

**Pull request merged**